features/levering-koopsom.feature

```diff
@@ -0,0 +1,152 @@
+Functionaliteit: levering koopsom
+
```

Comment:
ik denk dat het duidelijk is om naast de scenario's de functionaliteit eerst te benoemen. Dan is het duidelijker waar de scenario's een illustratie van zijn.

Reply:
Aangepast